### PR TITLE
Transition from atomic-dataset config option to tardis-refdata

### PR DESCRIPTION
**Pull request by vg3095**
This PR replaces `atomic-dataset` config option with `tardis-refdata`. Now `tardis-refdata` should point to Tardis reference folder. [Repository Link](https://github.com/tardis-sn/tardis-refdata)

### CODE REVIEW

**Comment by vg3095** [comment]
@wkerzendorf @yeganer You can review this PR.

@wkerzendorf Can you merge PR[#2](https://github.com/tardis-sn/tardis-refdata/pull/2) in `tardis-refdata` repository.

File: .travis.yml [hunk 1]
```diff
@@ -20,28 +20,29 @@ env:
         - ASTROPY_USE_SYSTEM_PYTEST=1
         - SETUP_CMD='test'
         - TEST_MODE='normal'
-        - ATOM_DATA_URL='https://github.com/tardis-sn/tardis-atomdata/raw/master/kurucz_cd23_chianti_H_He.h5.zip'
+        - TARDIS_REF_DATA_URL='https://github.com/tardis-sn/tardis-refdata.git'
+        - TARDIS_REF_VERSION='0.2'
```
Comment (wkerzendorf) [inline .travis.yml]: please no version for this.

File: tardis/plasma/tests/test_plasmas_full.py [hunk 2]
```diff
@@ -22,27 +22,23 @@ def plasma_compare_data_fname():
...
-                    reason='--atomic_database was not specified')
 class TestPlasmas():
     @classmethod
     @pytest.fixture(scope="class", autouse=True)
-    def setup(self):
-        self.atom_data_filename = os.path.expanduser(os.path.expandvars(
-            pytest.config.getvalue('atomic-dataset')))
-        assert os.path.exists(self.atom_data_filename), ("{0} atomic datafiles"
+    def setup(self, atomic_data_fname):
+        assert os.path.exists(atomic_data_fname), ("{0} atomic datafiles"
```
Comment (yeganer) [inline tardis/plasma/tests/test_plasmas_full.py]: Please move the check to the fixture

**Comment by vg3095** [comment]
@wkerzendorf @yeganer I have updated this PR . 
Integration tests are also running.

**Pull request merged**